### PR TITLE
Edx video pipeline

### PR DESCRIPTION
**Pull request by blarghmatey**
#### What are the relevant tickets?
#751 and #752 

#### What's this PR do?
Adds configs and scripts for building the edX video pipeline

#### Any background context you want to provide?
For the xPro deployment we need to be able to transcode and host our own videos

### CODE REVIEW

**Comment by blarghmatey** [comment]
@markbreedlove this might be useful to look over as a way of getting familiar with how this repo is organized

File: salt/environment_settings.yml [hunk 1]
```diff
@@ -123,12 +124,14 @@ environments:
       rabbitmq:
         role_prefixes:
           - edxapp
-          - ecommerce
```
Comment (shaidar) [inline salt/environment_settings.yml]: Why was this removed?

File: salt/environment_settings.yml [hunk 2]
```diff
@@ -362,7 +388,7 @@ environments:
           db_instance_class: db.t2.small
           allocated_storage: 100
           public_access: False
-          engine_version: '10.1'
+p          engine_version: '10.1'
```
Comment (shaidar) [inline salt/environment_settings.yml]: Typo?

File: salt/environment_settings.yml [hunk 3]
```diff
@@ -163,7 +166,22 @@ environments:
...
+          - veda-xpro-qa.mitx.mit.edu
+        instances:
+          edx-video-pipeline:
+            number: 1
+            type: t3.medium
+            security_groups:
+              - edx
+          edx-video-worker:
+            number: 2
+            type: t3.large
```
Comment (shaidar) [inline salt/environment_settings.yml]: Are those numbers and type needed for qa?

File: salt/environment_settings.yml (hunk 1 above)
Comment (blarghmatey) [reply salt/environment_settings.yml]: Because we aren't going to be using the ecommerce app any more

File: salt/environment_settings.yml (hunk 2 above)
Comment (blarghmatey) [reply salt/environment_settings.yml]: Yup...

File: salt/environment_settings.yml (hunk 3 above)
Comment (blarghmatey) [reply salt/environment_settings.yml]: I'm not sure yet, but given that the workers are actually running the transcode I would assume the answer to be yes.

**Comment by shaidar** [comment]
:thumbsup:

**Comment by shaidar** [comment]
Looks good to me
